lib/browser/api/menu-item.ts

```diff
@@ -43,6 +43,14 @@ const MenuItem = function (this: any, options: any) {
 
   this.overrideReadOnlyProperty('commandId', ++nextCommandId);
 
+  Object.defineProperty(this, 'userAccelerator', {
+    get: () => {
+      if (!this.menu) return null;
+      return this.menu._getUserAcceleratorAt(this.commandId);
```

Comment:
Let's have this not fail when called on non-macOS, it can just return null safely I think.